bundles/framework/layerselection2/Flyout.js

```diff
@@ -270,6 +273,15 @@ Oskari.clazz.define('Oskari.mapframework.bundle.layerselection2.Flyout',
                     sandbox.request(me.instance.getName(), request);
                     return false;
                 });
+            } else if (sandbox.hasHandler('DimensionChangeRequest')) {
```

Comment:
We need to have DimensionChangeRequest under Oskari-frontend instead of an app-repository to justify this.

Reply:
How about using layer.addTool() for adding the link?

Reply:
Worked on this in different branch and created PR https://github.com/oskariorg/oskari-frontend/pull/916. Closing this PR.